.github/workflows/release.yml

```diff
@@ -4,14 +4,18 @@
 name: Release
 on:
   release:
-    types: [prereleased, released]
```

Comment:
Only run this on release. Otherwise on a pre-release the plugin would also be published to the marketplace